### ISSUE TITLE
Accessibility Options/Onscreen Keyboard at Login Screen

### ISSUE THREAD

**Issue by 4rm** [twocanoes/xcreds#388]
I'd like to enable the onscreen keyboard at the login screen, but I'm unable to launch the accessibility options with Cmd-Opt-F5. Sometimes the accessibility options briefly flash after hitting the key combination, but then disappear right after. Possibly the window order is leveling it below XCreds.
